wasm-chain-integration/src/v1/types.rs

```diff
@@ -416,6 +422,11 @@ impl Output for ImportFunc {
                 CommonFunc::StateEntrySize => 20,
                 CommonFunc::StateEntryResize => 21,
                 CommonFunc::WriteOutput => 22,
+                CommonFunc::VerifyEd25519 => 32,
```

Comment:
Why the gap? To add more state-related host functions near each other?

Reply:
there is actually no gap. The Init/Receive functions are in the 22-31 range.